tests_integration/testing_utils/assertions/test_result_assertions.py

```diff
@@ -40,3 +42,11 @@ def assert_that_result_is_successful_and_done_is_on_output(
     assert_that_result_is_successful(result)
     assert result.output is not None
     assert json.loads(result.output) == {"message": "Done"}
+
+
+def _assert_that_no_error_logs(caplog):
+    error_logs = [
+        record.message for record in caplog.records if record.levelname == "ERROR"
+    ]
+    assert error_logs == []
+    caplog.clear()
```

Comment:
IMO we should change test's state as part of the assertion.

Reply:
Removed